calicoctl/commands/node/diags.go

```diff
@@ -235,3 +243,39 @@ func writeDiags(cmds diagCmd, dir string) {
 		log.Errorf("Error writing diags to file: %s\n", err)
 	}
 }
+
+func walkIPTables(file string, command string)([]byte) {
+
+	tablesFile, err := os.Open(file)
+	if err != nil {
+		fmt.Printf("Failed to open: %s\n", file)
```

Comment:
I think we need to return if we hit one of these errors, otherwise we'll try to continue below.